src/java.base/share/classes/jdk/internal/foreign/CABI.java

```diff
@@ -91,12 +94,6 @@ private static CABI computeCurrent() {
                 if (OperatingSystem.isLinux()) {
                     return LINUX_RISCV_64;
                 }
-            } else if (arch.startsWith("ppc64")) {
-                if (OperatingSystem.isLinux()) {
-                    return SYS_V_PPC_64LE;
-                } else {
-                    return AIX_PPC_64;
-                }
             } else if (arch.equals("s390x") && OperatingSystem.isLinux()) {
                 return SYS_V_S390X;
             }
```

Comment:
While you're here, what do you think of tweaking this to follow the pattern for other architectures, like this?
```java
            } else if (arch.equals("s390x")) {
                if (OperatingSystem.isLinux()) {
                    return SYS_V_S390X;
                }
            }
```

Reply:
Fixed as it make sense to me in this way.